plugins/offboard/offboard.h

```diff
@@ -91,7 +91,12 @@ class Offboard : public PluginBase
         float yawspeed_deg_s; /**< @brief Yaw angular rate in degrees/second (positive for clock-wise
                                    looking from above). */
     };
-
+    
+    /**
+     * @brief Type for altitude commands.
+     */
+    typedef float Altitude;
```

Comment:
Please specify if this is absolute or relative (to home) altitude and the unit.

Reply:
I don't think this typedef is needed to be consistent with the codebase.
```
float rel_altitude_m;
```
is fine I guess.

Reply:
Considering these comments I suggest that I remove de type definition in offboard.h; in offboard_impl.h (line 58) I will simply use float _rel_altitude_m. The *.cpp files will then set _rel_altitude_m accordingly.

Reply:
I observed there's no unit in set method, but in get method. For eg [set and get](https://github.com/dronecore/DroneCore/blob/develop/plugins/action/action.h#L250-L271) examples.